test/plugin/base_test.rb

```diff
@@ -1608,8 +1660,16 @@ def verify_json_log_entries(n, params, payload_type = 'textPayload')
         labels ||= request['labels']
         labels.merge!(entry['labels'] || {})
 
-        assert_equal \
-          "projects/#{params[:project_id]}/logs/#{params[:log_name]}", log_name
+        # For multi tag requests, each log name should match the index.
+        if params[:log_name] == 'multi'
```

Comment:
I would just check that `params[:log_name]` is non-`nil` and check for the constant log name, and otherwise do this in the block you pass to `verify_log_entries`.

Reply:
Good idea. Changed.